.flowconfig

```diff
@@ -1,7 +1,5 @@
 [ignore]
 
-.*/node_modules/fbjs/*
```

Comment:
Any reason why this is removed? It's failing with `"flow-bin": "^0.38.0",` for me.